src/components/FileMetadata/index.spec.tsx

```diff
@@ -16,7 +16,7 @@ import FileMetadata from '.';
 describe(__filename, () => {
   const _getVersionFile = (props = {}) => {
     const version = fakeVersion;
-    const store = createStoreWithVersion(version);
+    const store = createStoreWithVersion({ makeCurrent: true });
```

Comment:
@kumar303 Do we need to change it here as well?

Reply:
Yes, I missed this one. It should change, too.

Imagine changing the default version object (changing an id, a name, a description). This should not break any tests. If the test is relying implicitly on a name or id in the default version, it might start failing.

Reply:
okay @kumar303 .I updated the patch.

Thanks! :)